src/projector.ts

```diff
@@ -29,9 +29,17 @@ export interface ProjectorOptions extends ParentMixinOptions<Child>, EventedOpti
 	 *
 	 * Only applies if `autoAttach` is `true`
 	 */
+	// FIXME Is this used? What about replace?
```

Comment:
well, there aren't boolean literals (yet, they just said they were going to add them to TS2 today), so the only way to be explicit about `autoAttach` without `boolean` would be to do `autoAttach: undefined` and even then, `strictNullChecks` in TS2 would disallow that.  It is nice to have the option to be explicit instead of implicit.